src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -101,12 +105,20 @@ void satcheck_minisat2_baset<T>::clear_interrupt()
 
 const std::string satcheck_minisat_no_simplifiert::solver_text()
 {
+#ifdef HAVE_MERGESAT
+  return "Mergesat (latest) without simplifier";
```

Comment:
Hard coded version number, even `"latest"` make me nervous.

Reply:
Is there a way to run `git describe` in the MergeSat directory during build time, and use the output of that?

Reply:
Have a look at the set up for `src/util/version.cpp` here:

https://github.com/diffblue/cbmc/blob/74963e7f1d2bd584eafe715d5ed3ece538863760/src/util/Makefile#L122